### DISCUSSION TITLE
Serverless backend

### DISCUSSION THREAD

**Discussion by emregency** [directus/directus#4010] [Feature Requests]
Just a idea/question,

Would that make sense to be able to deploy the backend separately to Edges or serverless to be able to cope with high throughput scenarios?

**Comment by rijkvanzanten** [comment]
You could, but you'd still have a centralized database between all those edges, so unless you can figure out a way to have multiple copies of the DB itself too in the edges that are synced properly, the benefits are negligible 🤔 

If you're doing a lot of public reads, I would recommend relying on caching + a CDN instead, so the public API response is cached in the edge network

**Reply by emregency** [reply]
That makes more sense indeed. 👍

**Reply by casualmatt** [reply]
now with the support of cockroachdb, and API extension it could be a valid idea to support serverless install.

**Reply by rijkvanzanten** [reply]
@casualmatt What do you mean by _serverless install_? "Serverless" nowadays can mean a lot of different things 🙂

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
